app/posts/apply-for-teacher-training/2022-05-13-degree-autocomplete-changes.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Degree autocomplete changes
+description: Why we made changes to the autocomplete
+date: 2022-05-13
+---
+
+![TODO](degree-autocomplete-changes.png "Old and new autocompletes")
+
+The section of our application where candidates enter their degree information had ‘autocomplete’ fields for degree type (BA, BSc, etc), university name and degree subject.
```

Comment:
This feels an abrupt start.

Reply:
@edwardhorsford have simplified and used a list, let me know what you think.